### PR TITLE
Add documentation to reflect identical and equal mutators

### PR DESCRIPTION
**Pull request by BackEndTea**
For https://github.com/infection/infection/pull/391

### CODE REVIEW

**Comment by borNfreee** [comment]
could you please rebase on master @BackEndTea ?

File: src/guide/profiles.md [hunk 1]
```diff
@@ -41,16 +41,18 @@ All profiles are prepended by an `@` and in snake case, while all mutators are i
...
 * `@function_signature`
+* `@identical` - Disabled by default
 * `@number`
 * `@operator`
 * `@regex`
 * `@return_value`
 * `@sort`
 * `@zero_iteration`
-* `@default` - This is the default profile, which currently contains all mutators, and is used if no mutator or profile is chosen
+* `@default` - The standard profile, including all other profiles/mutators unless stated otherwise.
```
Comment (sanmai) [inline src/guide/profiles.md]: This could be better... How about if we just change `all` to `most` and leave it at it?

**Comment by borNfreee** [comment]
@BackEndTea github shows that there are still 2 conflicted files here, could you please rebase?

However, I don't see conflicts in the diff, probably it's related to today's github issues

**Comment by borNfreee** [comment]
Thnk you @BackEndTea

**Pull request merged**